### ISSUE TITLE
Include recording of audio

### ISSUE THREAD

**Issue by linuxundich** [phw/peek#105]
SimpleScreenRecorder has an option to record and audio track together with the video. You can choose the audio system (Pulse, Jack or Alsa) and the source. It would be great if Peek would support recording PulseAudio.

![simplescreenrecorder](https://cloud.githubusercontent.com/assets/1716643/23877989/3a9c1fc4-0845-11e7-94c8-4b6d759fed24.png)

**Comment by phw** [comment]
Not sure about this, it is starting to get a bit off from the GIF recorder thing :) If I support this it will definitely only be available for WebM / MP4 recording, and it will stay simple to setup (a single toggle to turn it on / off, PulseAudio required).

**Comment by ciscorucinski** [comment]
This is much needed! This is by far the best screen recorder that I have found.  Would love to use this instead of anything else.

For some reason, SimpleScreenRecorder will not record the screen but will record the audio. Peek will record the screen but not the audio.

**Comment by phw** [comment]
@ciscorucinski Agreed, but unfortunately I will not find the time to work on this any time soon. Help appreciated.

**Comment by mario6097** [comment]
I understand that there is not audio recording with peek, correct?

**Comment by phw** [comment]
Currently not

**Comment by trusktr** [comment]
I love Peek! It's the best simple screen recorder for Linux. Very easy to use, and just works. If it recorded audio too, that'd be awesome!

**Comment by gitowiec** [comment]
Peek is great! I use it when I want to show someone problems in software. I would also could use the recording audio feature. I would like to record a video on commenting a document formatting, recording just clicks is not helpful enough because I had to mark specified timestamps and describe what is happening on the video in the text file. Please add this feature! :)

**Comment by sudofox** [comment]
Bump...I'm helping a friend with some music-related things and want to show them my progress!

**Comment by patatetom** [comment]
hi,

why not _simply_ allow peek to pass additional parameters to the ffmpeg backend ?
from the command line for example...
`peek --ffmpeg="here are my additional parameters that I want to pass to ffmpeg"`
this way I could _simply_ ask peek/ffmpeg to capture the sound from pulse...
`peel --ffmpeg="-f pulse  -ac 2  -i default"`

regards.

**Comment by dm17** [comment]
I actually though it did! Would be great! What are most people using?
